README.md

```diff
@@ -1,4 +1,13 @@
 American-Gut
 ============
 
-American Gut open-access data and IPython notebooks
+American Gut open-access code and IPython notebooks
+
+A note about Data
+---
+
+American Gut sequences and metadata are deposited in [The European Bioinformatics Institute](http://www.ebi.ac.uk/ena) as ERP012803.
+
+The latest OTU tables and precalculated diversity comparisons generated by the primary processing notebook set can be found at [ftp://ftp.microbio.me/AmericanGut/](ftp://ftp.microbio.me/AmericanGut/).
```

Comment:
Change the link to ftp://ftp.microbio.me/AmericanGut/latest so it always point to the latest data.